app/controllers/application_controller.rb

```diff
@@ -1,5 +1,27 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+
   protect_from_forgery with: :exception
+  helper_method :current_user, :is_current_user?
+
+  def current_user
+    User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    if !session[:user_id]
```

Comment:
Convention for a something? method with ? is to return boolean without side effects